action.yml

```diff
@@ -20,10 +20,17 @@ branding:
 runs:
   using: composite
   steps:
+    # Set up a non-EOL, cibuildwheel & pipx supported Python version
+    - uses: mayeut/setup-python@no-envvar
+      id: python
+      with:
+        python-version: "3.7 - 3.10"
```

Comment:
I went with a range to allow self-hosted runners not to download python 3.10 if they have a supported version in their tool cache but no python 3.10 yet.
We could however just say `3.10` if we prefer, self-hosted runners would download & install it just fine if missing from their tool cache.

Reply:
The range seems great to me. No download time, and a supported Python!